jobspec/parse_task.go

```diff
@@ -374,6 +374,7 @@ func parseTemplates(result *[]*api.Template, list *ast.ObjectList) error {
 			ChangeMode: helper.StringToPtr("restart"),
 			Splay:      helper.TimeToPtr(5 * time.Second),
 			Perms:      helper.StringToPtr("0644"),
+			VaultGrace: helper.TimeToPtr(0),
```

Comment:
Added default value in parser. Time.period can not be nil.
This allows for silent parsing when vault_grace is not set or set to 0. Warning when set to a non zero value.